src/app/components/stepOne/NewFollow.tsx

```diff
@@ -0,0 +1,59 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import { usePatientContext } from '@/app/context/patientContext';
+import style from './NewFollow.module.css'
+import { useStepsContext, STEPS_NAMES } from '@/app/context/stepsContext'
+import { CardActionArea } from '@mui/material';
+
+const AppointmentOptions = [
```

Comment:
It is convention to use capital first letter for components and classes in react projects - this should be "appointmentOptions"